compatibility/pom.xml

```diff
@@ -14,7 +14,7 @@
     <properties>
         <hamcrest.version>3.0</hamcrest.version>
         <jackson.version>2.20.0</jackson.version>
-        <junit-jupiter.version>5.13.4</junit-jupiter.version>
+        <junit-jupiter.version>6.0.0-RC3</junit-jupiter.version>
```

Comment:
Note to self: Make a separate branch to check 5.14 (any RC) against
 * main
 * against the changes in this branch